java/ql/lib/semmle/code/java/security/regexp/PolynomialReDoSQuery.qll

```diff
@@ -75,8 +76,7 @@ module PolynomialRedosConfig implements DataFlow::ConfigSig {
   }
 
   predicate isBarrier(DataFlow::Node node) {
-    node.getType() instanceof PrimitiveType or
-    node.getType() instanceof BoxedType or
+    node instanceof SimpleScalarSanitizer or
```

Comment:
The method in the class `PolynomialRedosConfig`? I am fine modifying the class if we want. I wasn't sure whether or not to since the class configuration is deprecated. But I can see the argument for it for completeness sake.

Reply:
Just mentioning it because you modified other deprecated configuration classes in the PR. Your call.